packages/sirius-web/frontend/sirius-web/package.json

```diff
@@ -37,10 +37,12 @@
     "@xstate/react": "3.0.0",
     "d3": "7.0.0",
     "elkjs": "0.8.2",
+    "export-to-csv": "1.3.0",
     "fontsource-roboto": "4.0.0",
```

Comment:
You don't have to add a dependency to `sirius-components-tables` ?

Reply:
already done in packages/tables/frontend/sirius-components-tables/package.json